agent.py

```diff
@@ -4,9 +4,13 @@
 from langchain.text_splitter import RecursiveCharacterTextSplitter

```

Comment:
Automated Code Review Suggestion:

Critical fix applied.
```suggestion
import os
```

Missing import statement for `os` module.